tools/editor/editor_node.h

```diff
@@ -704,12 +709,38 @@ class EditorNode : public Node {
 
 struct EditorProgress {
 
-	String task;
-	void step(const String& p_state, int p_step=-1,bool p_force_refresh=true) { EditorNode::progress_task_step(task,p_state,p_step,p_force_refresh); }
-	EditorProgress(const String& p_task,const String& p_label,int p_amount) { EditorNode::progress_add_task(p_task,p_label,p_amount); task=p_task; }
-	~EditorProgress() { EditorNode::progress_end_task(task); }
+        String task;
```

Comment:
This chunk was changed to use spaces instead of tabs, so it is shown as modified even though no code was changed.